firefox_ui_harness/runners/base.py

```diff
@@ -2,7 +2,16 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-default_prefs = {
+import os
+
+import mozinfo
+from marionette import BaseMarionetteTestRunner
+
+import firefox_ui_tests
+from ..testcase import FirefoxTestCase
+
+
+DEFAULT_PREFS = {
     'app.update.auto': False,
     'app.update.enabled': False,
```

Comment:
We have a problem here, given that this pref is not obeyed and always set to `True` for all tests! This would mean a restart test could upgrade the binary under test, which is totally forbidden. All this can be added to our discussion about which defaults preferences have higher priority. Here we definitely fail with the current approach. I will file a bug so we can get this fixed ASAP.